backend/deepchecks_monitoring/schema_models/alert_webhook.py

```diff
@@ -238,14 +238,14 @@ def prepare_request_parameters(self, alert: "Alert", settings: "Settings") -> t.
         model = t.cast("Model", check.model)
         alert_link = str(prepare_alert_link(alert=alert, deepchecks_host=settings.deployment_url))
 
-        if self.kind == WebhookKind.STANDART:
+        if self.kind == WebhookKind.STANDARD:
             return {
                 "url": self.http_url,
                 "method": self.http_method,
                 "headers": self.http_headers,
                 "json": {
                     "alert_id": alert.id,
-                    "alert_name": f"models/{model.name}/monitors/{monitor.name}",
+                    "alert_name": f"models: {model.name} monitors: {monitor.name}",
```

Comment:
maybe in singular form?
"model: x, monitor: y"

Reply:
yeap, fixed